### PR TITLE
Dockerfile: Add required packages to enable IPsec

### PR DESCRIPTION
**Pull request by markdgray**
Signed-off-by: Mark Gray <mark.d.gray@redhat.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -40,6 +40,7 @@ RUN INSTALL_PKGS=" \
 	libpcap iproute strace \
 	containernetworking-plugins \
 	tcpdump iputils \
+	openvswitch-ipsec libreswan \
 	" && \
 	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False $INSTALL_PKGS && \
 	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False "openvswitch2.13 == $ovsver" "openvswitch2.13-devel == $ovsver" "python3-openvswitch2.13 == $ovsver" && \
```
Comment (dcbw) [inline Dockerfile:46]: @markdgray actually I think you need to add "openvswitch2.13-ipsec == $ovsver" to this line, and remove the "openvswitch-ipsec" from line 43 above. That should make things work. -72 is available in CI now.

Comment (markdgray) [reply Dockerfile:46]: Thanks @dcbw, I was assuming the package name would be the same in RHEL and, of course, it is not. I will try this morning.

**Comment by dcbw** [comment]
/test e2e-ovn-hybrid-step-registry
msg=Error: Error creating VPC: VpcLimitExceeded: The maximum number of VPCs has been reached.

**Pull request merged**